ti/instrumentation/instrumentation.go

```diff
@@ -59,17 +59,18 @@ func getTestSelection(ctx context.Context, runner TestRunner, config *api.RunTes
 			config.RunOnlySelectedTests = false // TI selected all the tests to be run
 			return selection, moduleList
 		}
-		if lastSuccessfulCommitID == "" {
-			log.Infoln("Test Intelligence determined to run all the tests to bootstrap")
-			config.RunOnlySelectedTests = false // TI selected all the tests to be run
-			return selection, moduleList
-		}
-		log.Infoln("Using reference commit: ", lastSuccessfulCommitID)
-		files, err = GetChangedFilesPush(ctx, workspace, lastSuccessfulCommitID, tiConfig.GetSha(), log)
-		if err != nil {
-			log.Errorln("Unable to get changed files list. Running all the tests.", "error", err)
+		if lastSuccessfulCommitID != "" {
+			log.Infoln("Using reference commit: ", lastSuccessfulCommitID)
+			files, err = GetChangedFilesPush(ctx, workspace, lastSuccessfulCommitID, tiConfig.GetSha(), log)
+			if err != nil {
+				log.Errorln("Unable to get changed files list. Running all the tests.", "error", err)
+				config.RunOnlySelectedTests = false
+				return selection, moduleList
+			}
+		} else {
+			// select all tests. It should still go to ti-service to update stats. This is full run bootstrap case
+			log.Infoln("No reference commit found")
```

Comment:
@anurag-harness : Why did we remove the "Test intelligence determined to run all the tests to bootstrap" message? How will the user know why we are running all the tests with this change?

Reply:
It goes to TI Service which sets selection.SelectAll to true and hence the code goes to L97 where we print 
log.Infoln("Test Intelligence determined to run all the tests")